chunky/src/java/se/llbit/chunky/renderer/Renderer.java

```diff
@@ -66,7 +67,7 @@
   void withBufferedImage(Consumer<BitmapImage> bitmap);
 
   interface SampleBufferConsumer {
-    void accept(double[] samples, int width, int height);
+    void accept(SampleBuffer samples, int width, int height);
```

Comment:
The sample buffer already has a `width` and a `height` so this could be removed. Actually the entire `SampleBufferConsumer` could then be replaced with `Consumer<SampleBuffer>`